diff --git a/node_modules/@medusajs/medusa-oas-cli/dist/command-oas.js b/node_modules/@medusajs/medusa-oas-cli/dist/command-oas.js
index 15f3cbc..a72b297 100644
--- a/node_modules/@medusajs/medusa-oas-cli/dist/command-oas.js
+++ b/node_modules/@medusajs/medusa-oas-cli/dist/command-oas.js
@@ -159,7 +159,7 @@ async function getPublicOas(apiType) {
 }
 async function getOASFromPaths(additionalPaths = [], customBaseFile) {
     console.log(`🔵 Gathering custom OAS`);
-    const gen = await (0, swagger_inline_1.default)(additionalPaths, {
+    const gen = await (0, swagger_inline_1.default)(additionalPaths.map(path => path.replace(/\\/g, "/")), {
         base: customBaseFile !== null && customBaseFile !== void 0 ? customBaseFile : path.resolve(basePath, "oas", "default.oas.base.yaml"),
         format: ".json",
         logger: (log) => {
