diff --git a/node_modules/swagger-typescript-api/templates/base/http-clients/fetch-http-client.ejs b/node_modules/swagger-typescript-api/templates/base/http-clients/fetch-http-client.ejs
index a1db383..33dfb66 100644
--- a/node_modules/swagger-typescript-api/templates/base/http-clients/fetch-http-client.ejs
+++ b/node_modules/swagger-typescript-api/templates/base/http-clients/fetch-http-client.ejs
@@ -2,6 +2,8 @@
 const { apiConfig, generateResponses, config } = it;
 %>
 
+import qs from "qs";
+
 export type QueryParamsType = Record<string | number, any>;
 export type ResponseFormat = keyof Omit<Body, "body" | "bodyUsed">;
 
@@ -86,14 +88,7 @@ export class HttpClient<SecurityDataType = unknown> {
 
     protected toQueryString(rawQuery?: QueryParamsType): string {
         const query = rawQuery || {};
-        const keys = Object.keys(query).filter((key) => "undefined" !== typeof query[key]);
-        return keys
-                .map((key) =>
-                    Array.isArray(query[key])
-                    ? this.addArrayQueryParam(query, key)
-                    : this.addQueryParam(query, key),
-                )
-                .join("&");
+        return qs.stringify(query);
     }
 
     protected addQueryParams(rawQuery?: QueryParamsType): string {
